style.css

```diff
@@ -0,0 +1,78 @@
+/* style sheet the portfolio */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+.main_container {
+  background-color: aliceblue;
+  max-width: 375px;
+  height: 718px;
+  margin: auto;
+}
+
+body {
+  background-color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.inside_container {
+  background-color: #fff;
+  background-image: url("./assets/header-shapes\ mobile.png");
+  padding-top: 8px;
+  height: 718px;
+  max-width: 375px;
+  border-bottom-left-radius: 100px;
+}
+
+.main_header {
+  padding: 0 32px;
+}
```

Comment:
- [ ] Kindly remove the padding on the header to match the spacing on the rest of the pages